api/Element.json

```diff
@@ -6581,7 +6581,8 @@
             ],
             "chrome_android": [
               {
-                "version_added": "37"
+                "version_added": "37",
+                "notes": "Version 92 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Comment:
What do these notes mean, precisely? Only version 92 returns the promise? Do previous versions return it (e.g., 91 or earlier)? Will subsequent versions return it?

Reply:
```suggestion
                "notes": "From version 92 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```